WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift

```diff
@@ -13,6 +13,8 @@ class QuickStartChecklistViewController: UITableViewController {
         self.blog = blog
 
         startObservingForQuickStart()
+
+        WPAnalytics.track(.quickStartChecklistViewed)
```

Comment:
Nitpicky maybe, but since this is a `viewed` event it would make sense to bump it from `viewDidAppear`.  In theory we could `init` the VC but never show it.

Reply:
I was worried about generating extra stats by flipping tabs, etc, but I'm fine changing this 👍

Reply:
Its actually useful to bump it that way if we ever wanted to create funnels tracking behavior across tabs :)